assembly_instructions/quality_control/tourniquet quality inspection.md

```diff
@@ -0,0 +1,81 @@
+# Quality inspection of Glia Tourniquets
+This document is revision: **A**
+
+## Revision history
+
+|Revision|Changes from previous revision|
+|---|---|
+|A | Initial creation of document|
+
+## Purpose
+
+This document contains the quality inspection characteristics and methods of inspection. In other words, what things need to be inspected and how to inspect those things.
+
+## Inspection characteristics
+
+### IC-0010 (Sewing complete)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+All sewing must be complete. All loose threads must be removed.
+
+### IC-0020 (Round end of strap heat sealed)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+The round end of the strap must be heat sealed.
+
+![IC-0020A1 End of strap heat sealed](./quality_images/IC-0020A1.jpg)
+
+
+### IC-0030 (Rivets installed)
+|Method of inspection|Tools required for inspection|
+|---|---|
+|Visual inspection|None|
+
+Both rivets are installed and secure.
+
+![IC-0030A1 Rivets in place from top](./quality_images/IC-0030A1.jpg)
+![IC-0030A2 Rivets in place from bottom](./quality_images/IC-0030A2.jpg)
```

Comment:
Does the backing plate have damage around the hole on the left side?  Has the rivet stressed the plastic plate?  There should be a note in the instruction regarding the insertion of the rivets and applying the correct pressure.

Reply:
We can ship with this image and then ask for a revised one. Issue to be created for this.